rspirv/sr/module.rs

```diff
@@ -39,7 +39,7 @@ pub struct Function {
     /// Function parameters.
     pub parameters: Vec<Token<Type>>,
     /// Basic blocks in this function.
-    pub basic_blocks: Vec<BasicBlock>,
+    pub basic_blocks: Vec<Token<BasicBlock>>,
```

Comment:
I'd be tempted to just store the first and let users traverse it as the graph that it is, but it doesn't matter too much.

Reply:
Good point. I think that would make sense!